aspnetcore/fundamentals/servers/aspnet-core-module.md

```diff
@@ -51,8 +51,12 @@ This section provides an overview of the process for setting up an IIS server an
 
 ### Install ANCM
 
+The ANCM is installed in IIS on Windows Server and in IIS Express on Windows desktop operating systems. For servers, the ANCM is included in the [.NET Core Windows Server Hosting bundle](https://aka.ms/dotnetcore-2-windowshosting). For development machines, Visual Studio automatically installs the ANCM in IIS Express and in IIS if it's already installed on the machine.
```

Comment:
What if I use VS Code on my dev machine?

Reply:
Yeah ... I don't like that either. I use VSC, and I run the hosting bundle installer. It's kind'a covered by the topic sentence ...

> The ANCM is installed in IIS on Windows Server and in IIS Express on Windows desktop operating systems. 

... but the last sentence confounds the point. I'll fix this now, and I'll hit the rest of the little probs in the topic on #5490.